.pre-commit-config.yaml

```diff
@@ -0,0 +1,21 @@
+repos:
+
+- repo: local
+  hooks:
+  - id: jupytext_format
+    name: jupytext_format
+    entry: jupytext --from ipynb --pipe black --check flake8 --pre-commit
+    pass_filenames: false
+    language: python
```

Comment:
Oooh I didn't know that jupytext could also run linters on notebooks

Reply:
Yeah I found this gem, which makes it so much easier to remain consistent with my code.

Reply:
This is amazing. This should get spammed all over our slack!

Reply:
cool!